src/app/components/super-admin-login/super-admin-login.component.css

```diff
@@ -12,37 +11,71 @@ div {
   justify-content: center;
   color: rgb(19, 20, 20);
 }
+.mat-mdc-card-header {
+  justify-content: center;
+}
+.mat-icon.mat-icon.notranslate.material-icons.mat-ligature-font.mat-icon-no-color {
+  position: relative;
+  top: -10px !important ;
+}
 
-mat-card {
-  padding: 2rem;
-  border-radius: 8px;
-  box-shadow: 0 8px 16px rgba(0.1, 0.1, 0.1, 0.1);
+.mat-mdc-card.admin-login {
+  display: flex;
+  flex-direction: row;
   background-color: white;
-  width: 400px;
+  width: 800px;
   height: 450px;
+  box-shadow: 0 8px 16px rgba(0.1, 0.1, 0.1, 0.1);
   justify-content: center;
   align-items: center;
 }
-
-mat-form-field {
+.mat-mdc-form-field {
   margin-bottom: 15px;
   width: 350px;
 }
-mat-label {
-  margin-left: 25px;
-}
-mat-card-header {
+.mat-mdc-card-header {
   font-size: larger;
   margin-bottom: 40px;
 }
-mat-error {
-  font-weight: 600;
-}
 .input-icon {
   display: flex;
 }
 .link {
-  color: #3f51b5; /* Primary color */
+  color: #3f51b5;
   text-decoration: underline;
   cursor: pointer;
 }
+.image-section {
+  width: 400px;
+  height: 450px;
+  color: white;
+  font-weight: bold;
+  font-size: larger;
+  justify-content: center;
+}
+.image-section img {
+  width: 400px;
```

Comment:
dont give explicit width and height
width: 100%;
  height: 100%;
  object-fit: cover;
  object-position: left;